src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -372,6 +387,29 @@ class ResourceService(
     } else None
   }
 
+  private def validatePolicyNotInherited(policy: FullyQualifiedPolicyId, samRequestContext: SamRequestContext): IO[Option[ErrorReport]] = {
+    for {
+      children <- accessPolicyDAO.listResourceChildren(policy.resource, samRequestContext)
+      policyOpt <- accessPolicyDAO.loadPolicy(policy, samRequestContext)
+    } yield {
+      policyOpt.flatMap { policy =>
+        if (policyInheritedByChildren(policy.descendantPermissions, children)) {
```

Comment:
why are we preventing this case?

Reply:
According to the ticket, overwriting the policy should "Return 400 if policy is inherited."

Reply:
@dvoet is this still something we want to prevent? Or was the ticket description a relic of when you could reference inherited policies with a '~'?

Reply:
Scrapped this as the ticket description was out of date